src/components/Layout/StandardLayout.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Footer from './Footer/Footer'
+import Header from './Header/Header'
+
+// interface IStandardLayoutProps {
+//   name: string;
+// }
+
+const StandardLayout = (/*{ name }: IStandardLayoutProps*/ props: any) => {
```

Comment:
This part is good and I would like you to use interfaces for all props, name them without `I` so this should be `StandardLayoutProps`. Also never use `any` type on purpose :).